src/main/java/duke/tasks/Todo.java

```diff
@@ -0,0 +1,61 @@
+package duke.tasks;
+
+import java.util.ArrayList;
+
+import duke.Duke;
+import duke.exceptions.tasks.EmptyDescriptionException;
+
+/**
+ * Represents a Todo task in the Duke application.
+ * A Todo task is a task without any date/time attached to it.
+ */
+public class Todo extends Task {
+    /**
+     * Constructs a new Todo task with the specified description.
+     * The task is initially not completed.
+     *
+     * @param description The description of the Todo task.
+     * @throws EmptyDescriptionException If the description is empty.
+     */
+    public Todo(String description) throws EmptyDescriptionException {
+        super(description);
+    }
+
+    /**
+     * Constructs a new Todo task with the specified description and completion status.
+     *
+     * @param description The description of the Todo task.
+     * @param isDone The completion status of the task.
+     * @throws EmptyDescriptionException If the description is empty.
+     */
+    public Todo(String description, boolean isDone) throws EmptyDescriptionException {
```

Comment:
I like that all your boolean variables are named properly and clearly to show its functions :)

Reply:
Thank you :)